### DISCUSSION TITLE
Letting tablet drivers run in the background while the pen is touching the tablet, causes cursor seizure.

### DISCUSSION THREAD

**Discussion by PuroOfficial** [ppy/osu#22683] [Q&A]
### Type

Game behaviour

### Bug description

I was doing this live, and this really bothers me because is my first drawing tablet and I excepted no issues.
It didn't affect osu!stable only osu!lazer.

### Screenshots or videos

https://www.youtube.com/live/TOBVoFLyfu0?feature=share&t=980

### Version

2022.1228.0

### Logs

Forgot during live-stream.

**Comment by bdach** [comment]
I'm not sure what we can do here really. If there are multiple active drivers at any given time then there's nothing that can be done other than disabling one of them (either the one in-game via the "Tablet" checkbox, or the one out-of-game). Especially so that I see that the in-game one has a custom Y offset applied.
